sources/v1.5.0/v1.5/step_3_enable_charge_invariant_violation/0-features.move

```diff
@@ -0,0 +1,24 @@
+// Script hash: 42ee0542 
+// Modifying on-chain feature flags:
+// Enabled Features: [ChargeInvariantViolation]
+// Disabled Features: []
+//
+script {
+    use aptos_framework::aptos_governance;
+    use std::features;
+    use std::vector;
+
+    fun main(proposal_id: u64) {
+        let framework_signer = aptos_governance::resolve_multi_step_proposal(proposal_id, @0000000000000000000000000000000000000000000000000000000000000001, vector::empty<u8>());
+
+        let enabled_blob: vector<u64> = vector[
+            20,
+        ];
+
+        let disabled_blob: vector<u64> = vector[
+
+        ];
+
+        features::change_feature_flags(&framework_signer, enabled_blob, disabled_blob);
```

Comment:
@runtian-zhou I thought the release builder should always try to do a reconfiguration after feature flag changes?

Reply:
This doesnt appear to be the case for v1.4 either however?

https://github.com/aptos-foundation/mainnet-proposals/blob/main/sources/v1.4/enable_struct_constructor/0-features.move#L22C19-L22C39